string id	#Identificativo del nodo richiedenti il lock
---
bool ack		#Risposta che comunica se il lock Ã¨ stato assegnato
